coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -152,9 +168,29 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
             raise KeyError("Language {!r} is not defined for docstyle {!r}."
                            .format(language, docstyle))
 
+        def get_from_docstyle_settings(*args):
+            values = list()
+            for req_setting in args:
+                setting = dict(
+                    docstyle_settings.contents.items()).get(req_setting)
+                try:
+                    setting = setting.value
+                except AttributeError:
+                    setting = ""
+                values.append(setting)
+            return values
```

Comment:
but you do inside `get_from_docstyle_settings`:
```python
try:
    setting = setting.value
except AttributeError:
    setting = ""
```
I don't understand completely what goes into this function, so I'm a bit confused about it and I have the feeling that it is not really necessary^^ (like said, looks like some "get-it-or-get-default" function :D)